eng/Dependencies.props

```diff
@@ -40,6 +40,7 @@ and are generated based on the last package release.
     <LatestPackageReference Include="Microsoft.Extensions.DependencyInjection" />
     <LatestPackageReference Include="Microsoft.Extensions.DependencyModel" />
     <LatestPackageReference Include="Microsoft.Extensions.DiagnosticAdapter" />
+    <LatestPackageReference Include="Microsoft.Extensions.Diagnostics" />
```

Comment:
Don't you need the Microsoft.Extensions.Diagnostics.Abstraction too? or will the dependency be automatically picked?

Reply:
CC @joperezr